addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4621,8 +4633,12 @@ export class OdooEditor extends EventTarget {
      * Handle safe pasting of html or plain text into the editor.
      */
     _onPaste(ev) {
-        ev.preventDefault();
         const sel = this.document.getSelection();
+        const protectedElement = closestElement(sel.anchorNode, '[data-oe-protected]');
+        if (protectedElement && (protectedElement.dataset.oeProtected === 'true' || protectedElement.dataset.oeProtected === '')) {
```

Comment:
This seems awefully close to the `isProtected` function in `utils.js`. Are you sure you cannot just call isProtected ?